sdk/core/Azure.Core/api/Azure.Core.netstandard2.0.cs

```diff
@@ -194,6 +194,7 @@ public partial class RequestContext
         public RequestContext() { }
         public System.Threading.CancellationToken CancellationToken { get { throw null; } set { } }
         public Azure.ErrorOptions ErrorOptions { get { throw null; } set { } }
+        public void AddClassifier(int[] statusCodes, bool isError) { }
```

Comment:
Are we going to add support for error codes in the future?  How would I specify that a certain combination of status + error code is not an error, but all other uses of that status code are?

Azure's goal is to use status codes for general error handling _(i.e., 409 for any sort of conflict where two people are trying to party on the same blob)_ and status + error codes for specific error handling _(i.e., 409 + LeaseAlreadyPresent when trying to acquire a lease on a blob that's already leased)_.  It's really common for folks to want error code level precision when classifying _(i.e., if I get a 409 + LeaseAlreadyPresent it's fine, but all other 409s like ContainerDisabled show still throw because I'm not writing code to handle them)_.

Would it be possible to have a lambda from `HttpMessage` to `bool`?  This would give us enough flexibility to do things like https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/storage/Azure.Storage.Common/src/Shared/StorageResponseClassifier.cs#L52-L71 which is even more specific than status + error code.

Reply:
Addressed: https://github.com/Azure/azure-sdk-for-net/pull/26547/files#diff-d8fef9e5888142bd6d9a5398c2d61a54cde4d38e79fdd50a204650a1cf44abaaR197